Rakefile

```diff
@@ -5,7 +5,12 @@ task :default => ['test:all']
 
 namespace :test do
   desc "test all drivers"
-  task :all => [:mysql2, :postgresql, :sqlite]
+  begin
+    require 'mysql'
+    task :all => [:mysql, :mysql2, :postgresql, :sqlite]
+  rescue LoadError
+    task :all => [:mysql2, :postgresql, :sqlite]
+  end
```

Comment:
I dropped mysql support on tests. https://github.com/basecamp/marginalia/commit/e03788f4e8a90df5478fa920e4c765012be13eb5

I think we can remove the require. do you need it?

Reply:
I don't need it. I'll remove this change.